### PR TITLE
fix(command-deployment): Cleanup page

### PR DESCRIPTION
**Pull request by Jiralite**
Following the guide from the beginning, there are some changes that need to be made to this page.

- Important comment difference between guide and code sample regarding folders vs. files.
- Removed "bonus commands". They should be introduced if they are actually introduced in the guide.
- Code samples were not 1:1 with the implementation in prior sections. This may cause confusion.

### CODE REVIEW

File: code-samples/creating-your-bot/command-deployment/index.js [hunk 1]
```diff
@@ -23,16 +23,18 @@ for (const folder of commandFolders) {
 	}
 }
 
-client.once(Events.ClientReady, () => {
-	console.log('Ready!');
+client.once(Events.ClientReady, c => {
```
Comment (almostSouji) [inline code-samples/creating-your-bot/command-deployment/index.js:26]: did we really not call that client across the board, `c`, really? that's unfortunate

Comment (monbrey) [reply code-samples/creating-your-bot/command-deployment/index.js:26]: It's a tradeoff between shadowing the variable, and showing that the event emits the client instance. It becomes relevant when we do event handling as it's no longer shadowing, but isn't relevant at this point in the guide.

Comment (almostSouji) [reply code-samples/creating-your-bot/command-deployment/index.js:26]: if it has no purpose I personally don't think we need it here and can instead introduce and explain it once it does make sense to have.
On the other hand, I can see the idea behind introducing the fact here, but if we go with that approach, let's find something nicer than `c` to establish verbose and understandable variable names as convention and best practice.

Comment (Jiralite) [reply code-samples/creating-your-bot/command-deployment/index.js:26]: For TypeScript users, `c` here will be emitted as `Client<ready>` whereas the previously-defined `client` is of type `Client<boolean>`. This means you won't be able to access `client.application` even though it's definitely defined in the ready event, so you'll have to do a pointless `client.isReady()` check.

Comment (almostSouji) [reply code-samples/creating-your-bot/command-deployment/index.js:26]: sounds good! everything is better than single characters as variable names and explaining why this is useful (and for who) is def. a good idea

Comment (Jiralite) [reply code-samples/creating-your-bot/command-deployment/index.js:26]: I have opened #1528!

**Review by almostSouji — APPROVED** [review]

**Pull request merged**